### PR TITLE
Add dependencies object to package json if none is found in UI CLI scaffolds

### LINKED ISSUE #297: Error when generating a Svelte UI with CLI
Running `zk project <name>` produces the following error:

`TypeError: Cannot set properties of undefined (setting 'snarkyjs')`

### PR DESCRIPTION
**Pull request by ymekuria**
Fixes #297

### CODE REVIEW

File: src/lib/project.js [hunk 1]
```diff
@@ -88,7 +88,11 @@ async function project({ name, ui }) {
 
     // Add SnarkyJS as a dependency in the UI project.
     let pkgJson = fs.readJSONSync(path.join('ui', 'package.json'));
-    pkgJson.dependencies.snarkyjs = '0.*';
+    // Add dependencies object if none is found in the package.json because generated
+    // Svelte projects do not have dependencies included.
```
Comment (jasongitmail) [inline src/lib/project.js:92]: *SvelteKit (worth clarifying for future readers)

**Review by jasongitmail — APPROVED** [review]
Suggested some changes. This makes sense though--I know SvelteKit uses devDependencies only

File: src/lib/project.js [hunk 2]
```diff
@@ -88,7 +88,11 @@ async function project({ name, ui }) {
...
+    pkgJson.dependencies
```
Comment (jasongitmail) [inline src/lib/project.js:93]: Personal preference, but I find this more clear. Not a strong opinion

```js
if (!pkgJson.dependencies) pkgJson['dependencies'] = {}
pkgJson.dependencies.snarkyjs = '0.*';
```

Comment (ymekuria) [reply src/lib/project.js]: Agreed this is cleaner.

**Pull request merged**